libctru/include/3ds/svc.h

```diff
@@ -241,6 +241,30 @@ typedef struct {
 
 ///@}
 
+///@name Processes
+///@{
+
+/// Information on address space for process. All sizes are in pages (0x1000 bytes)
+typedef struct {
+	u8 name[8];		///< ASCII name of codeset
+	u16 unk1;
+	u16 unk2;
+	u32 unk3;
+	u32 text_addr;				///< .text start address
```

Comment:
This should be extracted to a separate struct:

```c
typedef struct
{
	u32 addr;
	u32 size;
} CodeSetSecInfo;
```
and it would be used like this:
```
CodeSetSecInfo code;
CodeSetSecInfo rodata;
CodeSetSecInfo data;
```